desktop-app/js/tinder-desktop.swipe.js

```diff
@@ -14,315 +106,165 @@
+      evt.preventDefault();
+      $scope.undo();
+    });
 
-      // randomize rotation
-      $timeout(function() {
-        $.each($scope.cards, function(idx, card) {
-          var $card = $(card);
-          var marginLeft = parseInt($card.css('margin-left'));
-          $card.css('margin-left', '-' + (Math.floor(Math.random()*((marginLeft+10)-(marginLeft-10)+1)+(marginLeft-10))) + 'px')
-              .css('transform', 'rotate(' + (Math.floor(Math.random()*(3+3+1)-3)) + 'deg)');
+    Mousetrap.bind('up', function(evt) {
+      evt.preventDefault();
+      var numberOfPhotos = $scope.allPeople[$scope.allPeople.length-1].photos.length;
+      var photoIndex = $scope.allPeople[$scope.allPeople.length-1].photoIndex;
+      
+      if(photoIndex == 0){
+        $scope.allPeople[$scope.allPeople.length-1].photoIndex = numberOfPhotos - 1
+      } else {
+        $scope.allPeople[$scope.allPeople.length-1].photoIndex += -1
+      }
+      loadImage($scope.allPeople[$scope.allPeople.length-1].photoIndex);
+    }); 
+
+    Mousetrap.bind('down', function(evt) {
+      evt.preventDefault();
+      
+      var numberOfPhotos = $scope.allPeople[$scope.allPeople.length-1].photos.length
+      var photoIndex = $scope.allPeople[$scope.allPeople.length-1].photoIndex
+
+      if(photoIndex == numberOfPhotos - 1){
+        $scope.allPeople[$scope.allPeople.length-1].photoIndex = 0
+      } else {
+        $scope.allPeople[$scope.allPeople.length-1].photoIndex += 1
+      }
+      loadImage($scope.allPeople[$scope.allPeople.length-1].photoIndex);
+    });
+    
+    function loadImage(i){
+      var card = $scope.cards[$scope.cards.length-1];
+      (function(index,c){
+      var image = new Image();
+      image.onload = function () {
+        $scope.$apply(function () {
+          card.style.backgroundImage = 'url("' + $scope.allPeople[$scope.allPeople.length-1].photos[index].processedFiles[0].url+ '")';
+          card.className = "in-deck";
         });
-      }, 0, false);
-    };
-
+      };
+      image.src = $scope.allPeople[$scope.allPeople.length-1].photos[index].processedFiles[0].url;
+      })(i,card)
+    }
+    
+    $scope.$on('$locationChangeStart', function(event, next, current) {
+      if($scope.apiInQueue){
+        API[$scope.apiInQueue.method]($scope.apiInQueue.user._id);
+      }
+    });
+    
     getPeople();
-
   });
-
-  //helper
-  var debounce = function(func, wait, immediate) {
-    var timeout;
-    return function() {
-      var context = this, args = arguments;
-      var later = function() {
-        timeout = null;
-        if (!immediate) func.apply(context, args);
-      };
-      var callNow = immediate && !timeout;
-      clearTimeout(timeout);
-      timeout = setTimeout(later, wait);
-      if (callNow) func.apply(context, args);
+  
+  
+  module.filter('bdayToAge', function() {
+    return function(bday) {
+      return moment.duration(moment().diff(moment(bday))).years();
```

Comment:
This is in tinder-desktop.common.js. Looks like the conflict was incorrectly resolved.